docs/fundamentals/aspire-sdk-templates.md

```diff
@@ -0,0 +1,174 @@
+---
+title: .NET Aspire templates
+description: Learn how to install the .NET Aspire templates, and how to use them to create new apps.
+ms.date: 11/08/2024
+zone_pivot_groups: dev-environment
+uid: dotnet/aspire/templates
+---
+
+# .NET Aspire templates
+
+There are a number of .NET Aspire project templates available to you. You can use these templates to create full .NET Aspire solutions, or add individual projects to existing .NET Aspire solutions.
+
+The .NET Aspire templates are available in the [📦 Aspire.ProjectTemplates](https://www.nuget.org/packages/Aspire.ProjectTemplates) NuGet package.
+
+## Install the .NET Aspire templates
+
+:::zone pivot="visual-studio"
+
+The .NET Aspire templates are installed automatically when you install Visual Studio 17.9 or later.
+
+:::zone-end
+:::zone pivot="vscode,dotnet-cli"
+
+To install these templates, use the [dotnet new install](/dotnet/core/tools/dotnet-new-install) command, passing in the `Aspire.ProjectTemplates` NuGet identifier.
+
+```dotnetcli
+dotnet new install Aspire.ProjectTemplates
+```
+
+To install a specific version, append the version number to the package name:
+
+```dotnetcli
+dotnet new install Aspire.ProjectTemplates::9.0.0
+```
+
+> [!TIP]
+> If you already have the .NET Aspire workload installed, you need to pass the `--force` flag to overwrite the existing templates. Feel free to uninstall the .NET Aspire workload.
+
+:::zone-end
+
+## List the .NET Aspire templates
+
+:::zone pivot="visual-studio"
+
+The .NET Aspire templates are installed automatically when you install Visual Studio 17.9 or later. To see what .NET Aspire templates are available, select **File** > **New** > **Project** in Visual Studio, and search for "Aspire" in the search bar (<kbd>Alt</kbd>+<kbd>S</kbd>). You'll see a list of available .NET Aspire project templates:
+
+:::image type="content" source="media/vs-create-dotnet-aspire-proj.png" alt-text="Visual Studio: Create new project and search for 'Aspire'." lightbox="media/vs-create-dotnet-aspire-proj.png":::
+
+:::zone-end
+:::zone pivot="vscode"
+
+To view the available templates in Visual Studio Code with the C# DevKit installed, select the **Create .NET Project** button when no folder is opened in the **Explorer** view:
+
+:::image type="content" source="media/vscode-create-dotnet-proj.png" alt-text="Visual Studio Code: Create .NET Project button." lightbox="media/vscode-create-dotnet-proj.png":::
+
+Then, search for "Aspire 9" in the search bar to see the available .NET Aspire project templates:
```

Comment:
We should double check, but I don't think this is accurate as now everything is in a single template, so I don't think "9" is on the name.

```suggestion
Then, search for "Aspire" in the search bar to see the available .NET Aspire project templates:
```